packages/polling-controller/src/PollingController.test.ts

```diff
@@ -211,6 +211,37 @@ describe('PollingController', () => {
       await Promise.resolve();
       expect(controller.executePoll).toHaveBeenCalledTimes(2);
     });
+    it('should poll for the same networkClientId and options', async () => {
```

Comment:
It looks like we're calling `startPollingByNetworkClientId` with the same network client ID but _different_ options. What are we testing here?

Reply:
we are testing that it can start/stop polling for __different__ options. mainnet + 0x1 address should be seperate polling than mainnet + 0x2 address

Reply:
Maybe we can also add a permuation -> **different** networkClientId and **same** options